.travis.yml

```diff
@@ -162,20 +164,27 @@ jobs:
         - |
           # Get a kubernetes context.
           (. bin/_gcp.sh ; get_k8s_ctx "$GCP_PROJECT" "$GCP_ZONE" "$GKE_CLUSTER")
+        - |
+          # Install conduit cli.
+          version="git-$(git rev-parse --short=8 HEAD)"
```

Comment:
tioli, you could reuse some of `bin/root-tag`, or in `_tag.sh` use `git_sha_head`, or `head_root_tag`.